src/esp/nav/PathFinder.cpp

```diff
@@ -1349,18 +1350,26 @@ bool PathFinder::Impl::findPathSetup(MultiGoalShortestPath& path,
   if (!path.pimpl_->endRefs.empty())
     return true;
 
+  int numValidPoints = 0;
   for (const auto& rqEnd : path.getRequestedEnds()) {
     dtPolyRef endRef = 0;
     vec3f pathEnd;
     std::tie(status, endRef, pathEnd) =
         projectToPoly(rqEnd, navQuery_.get(), filter_.get());
 
     if (status != DT_SUCCESS || endRef == 0) {
-      return false;
+      ESP_DEBUG() << "Can't project end-point to navmesh, skipping: " << rqEnd;
+    } else {
+      numValidPoints++;
     }
 
     path.pimpl_->endRefs.emplace_back(endRef);
     path.pimpl_->pathEnds.emplace_back(pathEnd);
+
+    if (numValidPoints == 0) {
```

Comment:
Hold on, won't this still abort if the FIRST validPoint is doesn't snap even if subsequent ones are valid? Are the paths sorted?

Reply:
If it is sorted, add a comment.

Reply:
Lol, thank you for catching this. Typo, the conditional should be outside the loop.